src/main/java/com/easypost/model/Error.java

```diff
@@ -1,101 +1,14 @@
 package com.easypost.model;
 
 import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
 public final class Error {
-    private String message;
-    private String code;
+    private @Setter String message;
+    private @Setter String code;
```

Comment:
Users shouldn't be able to set these? Only get them. Are these needed?

Reply:
Yes, we need setters of these two variables for CustomErrorDeserlizer

https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/ErrorDeserializer.java#L35-L36

Reply:
Can we remove the setters and instead set these values inside the `Error` constructor?

Reply:
If we make a private constructor then yes, actually since they are in the same package, I can just make it default access modifier so end user will not be able to use it.